### DISCUSSION TITLE
Turn off hits counting

### DISCUSSION THREAD

**Discussion by zizom01** [RetroAchievements/RAIntegration#951] [Ideas]
hi i would like to suggest adding a new option in the achievements editor to turn off hits visually and maybe use another way of identifying if a logic line is active maybe when hits are turned off the highlight option gets activated automatically. i am requesting this feature because i worked with big achievements in term of achievements length in PCSX2 and when i activate it on the achievement editor it slows down the menus and sometimes the game itself so much that they are unusable until i deactivate the achievement or close the editor and sometimes i want to observe the code behavior and use the menus to change save states. also i have a high end pc so this issue is not from my side

**Comment by LiquifiedSnow** [comment]
Potentially just changing the background of the line to green if true and red if not true.  Maybe the option to switch between depending on what you're doing.
